### ISSUE TITLE
Github page

### ISSUE THREAD

**Issue by elfenars** [elfenars/ruby-discord-bot#6]

**Comment by breakfast1** [comment]
I just gained access to Bootstrap Studio with a student license. I could try it out there. Do you have something in mind ?

**Comment by elfenars** [comment]
I think the studio version is just an app for better designs, or does it change anything?, also, what happens after the student version is over? do we need to purchase it?

I wouldn't mind in general, but if there's a restriction of using **only** the studio version during or after the licence expires, then I do mind.

**Comment by elfenars** [comment]
e

**Comment by breakfast1** [comment]
There are no restrictions. One export the project to readable html and then we can modify it like a 'normal' web page.

**Comment by elfenars** [comment]
Cool, no problems then :)


From: Tanguy Andreani <notifications@github.com> <notifications@github.com>
Reply: elfenars/ruby-discord-bot
<reply@reply.github.com>
<reply@reply.github.com>
Date: April 23, 2018 at 12:36:13
To: elfenars/ruby-discord-bot <ruby-discord-bot@noreply.github.com>
<ruby-discord-bot@noreply.github.com>
CC: Feña Agar <ferliagno@gmail.com> <ferliagno@gmail.com>, State change
<state_change@noreply.github.com> <state_change@noreply.github.com>
Subject:  Re: [elfenars/ruby-discord-bot] Github page (#6)

There are no restrictions. One export the project to readable html and then

**Comment by breakfast1** [comment]
So do you have something in mind ?

**Comment by elfenars** [comment]
Not really, something simple with an intro and some docs on how to use and develop should do.
